prow/plugins/config.go

```diff
@@ -969,16 +969,16 @@ func validateConfigUpdater(updater *ConfigUpdater) error {
 	configMapKeys := map[ConfigMapID]sets.String{}
 	for file, config := range updater.Maps {
 		for cluster, namespaces := range config.Clusters {
+			if files.Has(file) {
+				return fmt.Errorf("file %s listed more than once in config updater config", file)
```

Comment:
`file` is a key from a map ... I think we can even remove the relevant checking

Reply:
Was this check existing before your PR? What was it doing then? Are we removing that?